source/_integrations/command_line.markdown

```diff
@@ -18,66 +21,7 @@ ha_integration_type: integration
 
 The `command_line` binary sensor platform issues specific commands to get data.
 
-## Configuration
-
-To use your Command binary sensor in your installation, add the following to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-binary_sensor:
-  - platform: command_line
-    command: "cat /proc/sys/net/ipv4/ip_forward"
-```
-
-<div class='note'>
-
-It's highly recommended to enclose the command in single quotes `'` as it ensures all characters can be used in the command and reduces the risk of unintentional escaping. To include a single quote in a command enclosed in single quotes, double it: `''`.
-
-</div>
-
-{% configuration %}
-command:
-  description: The action to take to get the value.
-  required: true
-  type: string
-name:
-  description: Let you overwrite the name of the device.
-  required: false
-  type: string
-  default: "*name* from the device"
-device_class:
-  description: Sets the [class of the device](/integrations/binary_sensor/), changing the device state and icon that is displayed on the frontend.
-  required: false
-  type: string
-payload_on:
-  description: The payload that represents enabled state.
-  required: false
-  type: string
-  default: 'ON'
-payload_off:
-  description: The payload that represents disabled state.
-  required: false
-  type: string
-  default: 'OFF'
-value_template:
-  description: Defines a [template](/docs/configuration/templating/#processing-incoming-data) to extract a value from the payload.
-  required: false
-  type: string
-scan_interval:
-  description: Defines number of seconds for polling interval.
-  required: false
-  type: integer
-  default: 60
-command_timeout:
-  description: Defines number of seconds for command timeout.
-  required: false
-  type: integer
-  default: 15
-unique_id:
-  description: An ID that uniquely identifies this binary sensor. Set this to a unique value to allow customization through the UI.
-  required: false
-  type: string
-{% endconfiguration %}
+{% include integrations/config_flow.md %}
```

Comment:
This integration does not connect to  a device or service, but instead defines its own protocol.  I would expect the YAML configuration to be remained  in that case.